wp-content/themes/core/components/blocks/spacer/block-editor.pcss

```diff
@@ -0,0 +1,22 @@
+
+/* -----------------------------------------------------------------------------
+ *
+ * Block Editor
+ *
+ * Adding admin styles for the Spacer Block.
+ *
+ * ----------------------------------------------------------------------------- */
+
+.wp-block[data-type^="acf/spacer"] {
+	.b-spacer {
+		border: 1px dashed #c8c8c8;
+
+		&:before {
+			content: '[SPACER]';
```

Comment:
It might be worth getting this out of the CSS and having it be part of the markup that is only visible in the editor and then display none'd in the FE? Thinking about the need for supporting other languages primarily.

Reply:
I'm not sure on this. I realize that it's text that should be translated but it's sole purpose is visualization so that editors know that the block even exists in that space. Having extra markup on the front end just for that seems unnecessary. 

Maybe instead of copy it should just be something like [------------] , or even an emoji :space_invader:

Reply:
Why not mimic the UX that already exists in the core spacer block and give it a background color? http://p.tri.be/RuMOsc

Reply:
For sure, totally get that. Thinking of like WP admin also being in other languages. Probably a vote for mimicking core and/or removing the text piece as you noted.

Reply:
So I tested core spacer and the issue is that the background color goes away after focus is removed. The issue the content editors were having was they were losing track of the block and couldn't figure out if there was actually anything there. Mimicking in this case doesn't solve that.

Reply:
Does the [List View](http://p.tri.be/v7Ojfx) or changing to [Select Mode](http://p.tri.be/DCQBIA) solve that issue? 

It seems like this is a training issue. It would cause more confusion to show something that isn't going to display on the FE causing users to try to remove the extra design items because they don't realize that this doesn't work the same as other blocks.

Reply:
While I agree with you, I removed the text from outputting and now there's just a top and bottom dashed border. I'm going to leave the rest up to the the design team since this was their scope